tests/mock_network/node_ageing.rs

```diff
@@ -245,6 +228,24 @@ fn remove_node_from_prefix(nodes: &mut Vec<TestNode>, prefix: &Prefix<XorName>)
     nodes.remove(index)
 }
 
+// Make the given section churn the given number of times, while maintaining the section size in
+// the interval that allow demoting/relocating a node at each step.
+fn section_churn_allowing_relocate(
+    count: usize,
+    network: &Network,
+    nodes: &mut Vec<TestNode>,
+    prefix: &Prefix<XorName>,
+) {
+    // Keep the section size such that relocations can happen but splits can't.
+    // We need NETWORK_PARAMS.elder_size + 1 excluding relocating node for it to be demoted.
+    let min_size = (NETWORK_PARAMS.elder_size + 1) + 1;
```

Comment:
Minor: why not `NETWORK_PARAMS.elder_size + 2` ?

Reply:
it seemed more explicit, we want NETWORK_PARAMS.elder_size + 1  plus 1 relocating node.
But I can change it if that does not make sense

Reply:
Nah, it's fine.